### ISSUE TITLE
RF storage crashes game

### ISSUE THREAD

**Issue by GHOziLINA** [TeamBR-Modding/NeoTech#244]
What Version of NeoTech are you using: 2.4.1

If possible, did you break and replace the block: game crashed after placing block can't open again
rf storage cell

Please Pastebin.com the crash Report and put link here: http://pastebin.com/Fp6bKZ7c

updated to 2.4.3 and the same thing still http://pastebin.com/FZeZ1KQG

**Comment by pauljoda** [comment]
Are you using the latest bookshelf for 1.8?

**Comment by GHOziLINA** [comment]
yes 

```
On Wednesday, March 23, 2016 7:27 PM, Paul Davis <notifications@github.com> wrote:
```

 Are you using the latest bookshelf for 1.8?—

**Comment by GHOziLINA** [comment]
Bookshelf-API-1.8.9-2.3.7 

```
On Wednesday, March 23, 2016 7:27 PM, Paul Davis <notifications@github.com> wrote:
```

 Are you using the latest bookshelf for 1.8?—

**Comment by Dyonovan** [comment]
You have a mod that is using the non-official version of RF API and it is overwriting ours. CoFH has released and official RF API for 1.8.9 and that mod needs to update to that as they are not compatible

**Comment by GHOziLINA** [comment]
can you please send me a link

```
On Wednesday, March 23, 2016 9:57 PM, Dyonovan <notifications@github.com> wrote:
```

 You have a mod that is using the non-official version of RF API and it is overwriting ours. CoFH has released and official RF API for 1.8.9 and that mod needs to update to that as they are not compatible—

**Comment by GHOziLINA** [comment]
can you please send me a link
Thanks

```
On Wednesday, March 23, 2016 9:57 PM, Dyonovan <notifications@github.com> wrote:
```

 Closed #244.—
